### PR TITLE
Add Bitcoin Core 19055

### PR DESCRIPTION
**Pull request by adamjonas**

### CODE REVIEW

File: _posts/en/newsletters/2021-01-13-newsletter.md [hunk 1]
```diff
@@ -138,7 +138,18 @@ repo], [Hardware Wallet Interface (HWI)][hwi repo],
 
 - [Bitcoin Core #18077][] net: Add NAT-PMP port forwarding support FIXME:Xeyko
 
-- [Bitcoin Core #19055][] muhash FIXME:adamjonas
+- [Bitcoin Core #19055][] adds the Muhash algorithm and uses it to calculate
+  the UTXO set hash in `gettxoutsetinfo`.  As covered in
```
Comment (harding) [inline _posts/en/newsletters/2021-01-13-newsletter.md]: Are you sure it gets used?  I thought this was only adding the function, with 19145 actually adding it to the RPC.

**Review by harding — CHANGES_REQUESTED** [review]
You might also want to link to the AssumeUTXO topic, e.g. "...creating an index for UTXO set statistics, which will make it easy to verify [assumeUTXO][topic assumeutxo] archives, meta PR ..."

File: _posts/en/newsletters/2021-01-13-newsletter.md (hunk 1 above)
Comment (adamjonas) [reply _posts/en/newsletters/2021-01-13-newsletter.md]: Whoops. Updated.

**Comment by adamjonas** [comment]
Added the ref to assumeutxo - I was trying to cut down on the repetition from previous coverage, but it makes sense to include. Thanks!

**Comment by harding** [comment]
Thanks @adamjonas !

**Pull request merged**